package.json

```diff
@@ -22,9 +22,11 @@
     "pretest": "yarn validate && shx rm -rf coverage",
     "test": "yarn jest",
     "test:watch": "yarn test --watch",
-    "prebuild": "yarn test --ci && shx rm -rf dist",
-    "build": "NODE_ENV=production yarn build:js && yarn build:sass",
-    "build:js": "babel src --config-file ./config/babel.config.js --out-dir dist/js --only 'src/**/*.js' --ignore 'src/jest,src/cosmos,**/tests/**,**/fixtures/**'",
+    "build": "yarn build:babel && yarn build:sass",
```

Comment:
should `NODE_ENV=production` and `&& shx rm -rf dist` be preserved?

Reply:
`NODE_ENV=production` doesn't matter. I added back the `prebuild` step, though.